pandas/io/gbq.py

```diff
@@ -148,6 +151,43 @@ def get_credentials(self):
 
         return credentials
 
+    def get_credentials_for_service_account(self):
```

Comment:
there is simple reason - to name small part of code. function name is self explaining